SnakesAndLadders/Classes/Game.cs

```diff
@@ -0,0 +1,85 @@
+﻿namespace SnakesAndLadders.Classes
+{
+    public class Game
+    {
+        public bool IsFinished { get; set; } = false;
+
+        public List<Player> Players { get; set; }
+
+        private readonly GameCollections _gameCollections;
+
+
+        public Game()
+        {
+            Players = new List<Player>();
```

Comment:
Inconsistencia de inicializacion. El is finished lo inicializas arriba pero este aqui. Siempre consistente, en uno o en otro pero consistente